websocketOutportDriver/websocketOutportDriverNodePart.go

```diff
@@ -0,0 +1,202 @@
+package websocketOutportDriver
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	outportSenderData "github.com/ElrondNetwork/elrond-go-core/websocketOutportDriver/data"
+)
+
+const sleepBetweenRetries = 200 * time.Millisecond
+
+// WebsocketOutportDriverNodePartArgs holds the arguments needed for creating a new websocketOutportDriverNodePart
+type WebsocketOutportDriverNodePartArgs struct {
+	Enabled                  bool
+	Marshalizer              marshal.Marshalizer
```

Comment:
So here we rely on the fact that whoever sets up a node with outport driver, all its clients are going to use the same marshallizer?

Reply:
yes, that's true